2020/11/26 16:45:50 [error] 272#0: *4987 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 262144 bytes) in /Users/Mick/Code/primio-backend-2020/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 776PHP message: PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 262144 bytes) in Unknown on line 0" while reading response header from upstream, client: 127.0.0.1, server: , request: "GET /api/v1/cards/1 HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/11/26 16:46:21 [error] 272#0: *4987 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 262144 bytes) in /Users/Mick/Code/primio-backend-2020/vendor/laravel/framework/src/Illuminate/Container/Container.php on line 776PHP message: PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 1052672 bytes) in /Users/Mick/Code/primio-backend-2020/vendor/guzzlehttp/psr7/src/Stream.php on line 228" while reading response header from upstream, client: 127.0.0.1, server: , request: "GET /api/v1/cards/1 HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:37:26 [crit] 255#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:37:28 [crit] 255#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:37:28 [crit] 255#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test", referrer: "http://primio-backend-2020.test/nova"
2020/12/10 15:37:36 [crit] 255#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:37:37 [crit] 255#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:38:48 [crit] 257#0: *8 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:38:49 [crit] 257#0: *8 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 15:41:35 [crit] 3086#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:44:31 [crit] 3090#0: *3 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:48:26 [crit] 3092#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:52:26 [crit] 3090#0: *7 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:52:27 [crit] 3090#0: *7 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:52:46 [crit] 3090#0: *7 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:52:47 [crit] 3090#0: *7 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:52:59 [crit] 3090#0: *7 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:54:32 [crit] 260#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:54:32 [crit] 261#0: *3 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test", referrer: "http://primio-backend-2020.test/nova/dashboards/main"
2020/12/10 16:55:42 [crit] 267#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:56:05 [crit] 267#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:56:32 [crit] 267#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:56:57 [crit] 267#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:57:00 [crit] 267#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 16:57:01 [crit] 267#0: *5 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/10 17:06:50 [crit] 264#0: *12 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/11 13:28:00 [crit] 263#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "POST /nova/login HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test", referrer: "http://primio-backend-2020.test/nova/login"
2020/12/11 13:28:05 [crit] 263#0: *1 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "POST /nova/login HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test", referrer: "http://primio-backend-2020.test/nova/login"
2020/12/11 13:32:40 [crit] 262#0: *4 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/11 13:35:47 [crit] 264#0: *6 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /api/v1/user/1 HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/11 13:35:52 [crit] 267#0: *8 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/11 13:37:09 [crit] 267#0: *10 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /test HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "dir.test"
2020/12/11 13:37:09 [crit] 262#0: *12 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "dir.test", referrer: "http://dir.test/test"
2020/12/11 13:37:12 [crit] 267#0: *10 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /test.php HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "dir.test"
2020/12/11 13:37:12 [crit] 262#0: *12 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "dir.test", referrer: "http://dir.test/test.php"
2020/12/11 13:37:16 [crit] 260#0: *16 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2020/12/11 13:37:18 [crit] 260#0: *16 connect() to unix:/Users/Mick/.config/valet/valet.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: , request: "GET /nova/dashboards/main HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2021/01/26 14:17:56 [error] 264#0: *4839 upstream prematurely closed connection while reading response header from upstream, client: 127.0.0.1, server: , request: "GET /api/v1/test HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test"
2021/02/02 15:41:10 [alert] 263#0: *6940 writev() failed (41: Protocol wrong type for socket) while sending to client, client: 127.0.0.1, server: , request: "GET /files/26pcnGizEKwYZuBNmJTV9jNeGAOP7Z6I8qcckq1L.png HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock:", host: "primio-backend-2020.test", referrer: "http://primio-backend-2020.test/nova/resources/playbooks/27"
2021/06/04 05:28:00 [error] 282#0: *976 upstream timed out (60: Operation timed out) while reading response header from upstream, client: 127.0.0.1, server: , request: "POST /broadcasting/auth HTTP/1.1", upstream: "fastcgi://unix:/Users/Mick/.config/valet/valet.sock", host: "five-backend-2021.test"
